src/DotNetWorker.Core/Invocation/IFunctionExecutor.cs

```diff
@@ -1,12 +1,12 @@
-// Copyright (c) .NET Foundation. All rights reserved.
+﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 using System.Threading.Tasks;
 using Microsoft.Azure.Functions.Worker.Pipeline;
 
 namespace Microsoft.Azure.Functions.Worker.Invocation
 {
-    internal interface IFunctionExecutor
+    public interface IFunctionExecutor
```

Comment:
This was a temporary set of changes required to add a custom executor. Will remove, but we'll eventually need this.

Reply:
I noticed FunctionApp26 has an implementation of this ([DirectFunctionExecutor](https://github.com/kshyju/FunctionApp26/blob/fabiocav/tweaks/FunctionApp26/DirectFunctionExecutor.cs#L24)). Is our plan to add some source generator to generate that for the customer function?